apps/finance/app/src/App.js

```diff
@@ -99,65 +99,72 @@ class App extends React.Component {
-          {transactions.length > 0 && (
-            <SpacedBlock>
-              <Transfers transactions={transactions} tokens={tokens} />
-            </SpacedBlock>
-          )}
-          {balances.length === 0 &&
-            transactions.length === 0 && (
-              <EmptyScreen>
-                <EmptyStateCard
-                  icon={<img src={addFundsIcon} alt="" />}
-                  title="Add funds to your organization"
-                  text="There are no funds yet - add funds easily"
-                  actionText="Add funds"
-                  onActivate={this.handleNewTransferOpen}
-                />
-              </EmptyScreen>
+        <Main>
+          <AppView
+            appBar={
+              <AppBar
+                title="Finance"
+                endContent={
+                  <Button mode="strong" onClick={this.handleNewTransferOpen}>
+                    New Transfer
+                  </Button>
+                }
+              />
+            }
+          >
+            {balances.length > 0 && (
+              <SpacedBlock>
+                <Balances balances={balances} />
+              </SpacedBlock>
+            )}
+            {transactions.length > 0 && (
+              <SpacedBlock>
+                <Transfers transactions={transactions} tokens={tokens} />
+              </SpacedBlock>
             )}
-        </AppView>
-        <SidePanel
-          opened={newTransferOpened}
-          onClose={this.handleNewTransferClose}
-          title="New Transfer"
-        >
-          <NewTransferPanelContent
-            app={app}
+            {balances.length === 0 &&
+              transactions.length === 0 && (
+                <EmptyScreen>
+                  <EmptyStateCard
+                    icon={<img src={addFundsIcon} alt="" />}
+                    title="Add funds to your organization"
+                    text="There are no funds yet - add funds easily"
+                    actionText="Add funds"
+                    onActivate={this.handleNewTransferOpen}
+                  />
+                </EmptyScreen>
+              )}
+          </AppView>
+          <SidePanel
             opened={newTransferOpened}
-            tokens={tokens}
-            onWithdraw={this.handleWithdraw}
-            onDeposit={this.handleDeposit}
-            proxyAddress={proxyAddress}
-            userAccount={userAccount}
-          />
-        </SidePanel>
+            onClose={this.handleNewTransferClose}
+            title="New Transfer"
+          >
+            <NewTransferPanelContent
+              app={app}
+              opened={newTransferOpened}
+              tokens={tokens}
+              onWithdraw={this.handleWithdraw}
+              onDeposit={this.handleDeposit}
+              proxyAddress={proxyAddress}
+              userAccount={userAccount}
+            />
+          </SidePanel>
+        </Main>
       </PublicUrl.Provider>
     )
   }
 }
 
+const Main = styled.div`
+  height: 100vh;
```

Comment:
Just wanted to double check this doesn't run into the same problem we had with `vh` due to the banner. I think it shouldn't be a problem since this is would be inside the iframe?

Reply:
Yes exactly 👍